app/middleware/parameters.js

```diff
@@ -27,7 +27,7 @@ module.exports = function() {
 
     ctx._setParameters(params);
 
-    ctx.logger.info('[parameters]', _.omit(ctx.params, ...filterParameters));
+    ctx.logger.info('[parameters]', JSON.stringify(_.omit(ctx.params, ...filterParameters)));
```

Comment:
`ctx.logger.info('[parameters] %j', .omit(ctx.params, ...filterParameters));` will be better.